.github/workflows/publish-docs-manual.yml

```diff
@@ -49,18 +47,14 @@ jobs:
           rm -rf main
 
       - name: Generate docs
-        run: make -C docs
+        run: make -C docs docs clean-k0s
```

Comment:
`make -C docs docs`

Reply:
Without the explicit clean step, mike will actually try to commit the k0s binary to the docs folder. See #1499.